### DISCUSSION TITLE
Alfen - Eve Single Pro-line setup (managed by 3rd party with 4G-SIM)

### DISCUSSION THREAD

**Discussion by aapo1974** [lbbrhzn/ocpp#891] [Q&A]
I have a third party managed (with 4G-SIM) Alfen - Eve Single Pro-line home charger. It connected via RJ45 to my home network and local ip. I was able to connect to the charger via the MyEve -android app.

I installed OCPP on my HA with HACS. Should the Alfen automatically be discovered after initial OCPP setup? Or do I not have the right settings in the Alfen device?

EDIT: So I added Alfen Wallbox integration with HACS and try to set up. I have the ip but I don't have the username or password. May it is a dead end...? Any ideas?

**Comment by Hammertime84** [comment]
I had this same problem. to change the backoffice settings you need the owner password of your Alfen device. Usually your backoffice provider has this. You also need to have access to the ACE service installer (Windows) or the MyEve app.

My Alfen Single Pro Line can have 1 backoffice URL configured. So if you succeed in getting the owner password, you will break the backoffice connection when integrating with HA..
